MaxText/configs/base.yml

```diff
@@ -206,8 +206,12 @@ load_from_prefill_dir: False # If true, decode.py doesn't "prefill" but just rea
 prefill_cache_dir: "" # If set and load_from_prefill_dir, decode.py reads from directory. If set, decode.py writes to directory
 autoregressive_decode_assert: ""
 
+# For nsys profiler, pass the training command to nsys command
+# e.g. nsys profile -s none --force-overwrite true --capture-range=cudaProfilerApi --capture-range-end=stop {training command} 
+profiler: "" # Supported profiler: '', xprof, nsys
```

Comment:
"xprof" is an internal tool, we open source we need a different name, maybe xplane (is there a better name?)